tools/build-extension-ios.sh

```diff
@@ -6,7 +6,16 @@ SYMBOLS=$1
 SYMBOLS_FILE=$2
 COPY_PATH=$3
 
-DEPS=`cat "$SRCROOT/$PRODUCT_NAME.ios"`
+read SDK_MAJORVERSION SDK_MINORVERSION <<<${SDK_NAME//[^0-9]/ }
+
+if [[ $SDK_MAJORVERSION -ge 14 && "$PRODUCT_NAME" == "standalone" ]] ; then
```

Comment:
@livecodepanos I think this means you need to add this file for each of the externals. Perhaps check if it's there and fallback?